packages/tooling/fast-figma-plugin-msft/package.json

```diff
@@ -28,34 +30,28 @@
   "homepage": "https://www.fast.design/",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.12.13",
-    "@babel/preset-env": "^7.12.13",
-    "@figma/plugin-typings": "^1.41.1",
+    "@figma/plugin-typings": "^1.47.0",
     "@types/node": "^15.0.1",
-    "@types/react": "^16.9.17",
-    "@types/react-dom": "^16.9.4",
-    "babel-loader": "^8.2.5",
     "clean-webpack-plugin": "^4.0.0",
     "copy-webpack-plugin": "^10.2.4",
     "css-loader": "^6.7.1",
     "eslint-config-prettier": "^6.10.1",
     "html-webpack-inline-source-plugin": "^1.0.0-beta.2",
     "prettier": "2.0.2",
     "raw-loader": "^4.0.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
     "style-loader": "^3.3.1",
     "ts-loader": "^9.3.0",
     "typescript": "^4.7.0",
     "webpack": "^5.72.0",
     "webpack-bundle-analyzer": "^4.5.0",
     "webpack-cli": "^4.9.2"
   },
+  "peerDependencies": {
+    "@microsoft/fast-colors": "5.1.5",
+    "@microsoft/fast-element": "1.9.0",
+    "@microsoft/fast-foundation": "2.42.0"
+  },
   "dependencies": {
-    "@fluentui/web-components": "^2.2.1",
-    "@microsoft/fast-colors": "^5.3.1",
-    "@microsoft/fast-components": "^2.30.6",
-    "@microsoft/fast-element": "^1.0.0",
-    "@microsoft/fast-react-wrapper": "^1.0.0-alpha.3"
+    "@fluentui/web-components": "2.5.2"
```

Comment:
@bheston The latest version here is 2.5.3 (updated to use latest versions of all FAST). I think you'd need to be on latest as opposed to the pinned versions above to use it FWIW.

Reply:
Updated. I went back to the `^` version so it would pick up latest.